src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/personalrelationships/ContactPersonMappingApiMockServer.kt

```diff
@@ -236,6 +236,41 @@ class ContactPersonMappingApiMockServer(private val objectMapper: ObjectMapper)
       )
     }
   }
+  fun stubGetByDpsContactAddressIdOrNullAsNullFollowedByValue(
```

Comment:
needed because service will check if address has been created yet, which it won't to begin with and then  it will require the address to be present before it can create an address phone - hence the scenario